x/stake/keeper/key.go

```diff
@@ -84,14 +89,16 @@ func GetValidatorPowerRank(validator types.Validator, pool types.Pool) []byte {
 				append(heightBytes, counterBytes...)...)...)...)
 }
 
-// get the key for the accumulated update validators
+// get the key for the accumulated update validators.
+// The value at this key is of type stake/types.Validator
```

Comment:
I think it would be cool to normalize the text in the comment here to something a bit less verbose like:
```golang
// VALUE: type stake/types.Validator
```